sdk/exporter/exporter.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	"context"
+)
+
+// Syncer is a type for functions that receive a single trace span.
+//
+// The ExportSpan method is called synchronously.
```

Comment:
Might be worth adding a note that ExportSpan implementations should not block indefinitely

Reply:
Sure, though that was implied by the fact that the method is called synchronously.